apps/cli/src/domain/node.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
+
+import { Dependencies } from "./dependencies.js";
+import { unwrapOrLogError } from "./index.js";
+
+const ScriptName = z.string().brand<"ScriptName">();
+
+export const ScriptSchema = z.object({
```

Comment:
nitpick: variable names should be in camelCase, so this schema should be renamed `scriptSchema`